integrationTests/testProcessorNode.go

```diff
@@ -771,11 +771,22 @@ func (tpn *TestProcessorNode) initGenesisBlocks(args ArgTestProcessorNode) {
 			tpn.SmartContractParser,
 			tpn.EnableEpochs,
 		)
+
+		err := OnExecutedBlock(tpn)
+		if err != nil {
+			log.Error("tpn.initGenesisBlocks", "err", err)
```

Comment:
[globally in this file]
Perhaps in this file you could add extra message message like
```suggestion
			log.Error("tpn.initGenesisBlocks: OnExecutedBlock(tpn)", "err", err)
```